AISKU/Tests/TestFramework/TestClass.ts

```diff
@@ -80,24 +87,26 @@ class TestClass {
                                 nextTestStepTrigger.call(this);
                             }
                         } catch (e) {
-                            this._testCompleted();
+                            console.error("Failed: Unexpected Exception: " + e);
+                            this._testCompleted(true);
                             Assert.ok(false, e.toString());
-
+                            completed = true;
```

Comment:
Is this `completed` duplicated with `this._testCompleted`? 🤔

Reply:
No _testCompleted is a method, this is a flag to indicated that no additional steps should be processed.